containers/layouts/LayoutsSection.js

```diff
@@ -110,6 +118,23 @@ const LayoutsSection = () => {
                     <ViewModeRadios viewMode={ViewMode} onChange={handleChangeViewMode} loading={loadingViewMode} />
                 </Field>
             </Row>
+            <Row>
+                <Label>
```

Comment:
Since you defined an `id` for the toggle, you should use it on the `Label` component.
```suggestion
                <Label htmlFor="stickyLabelsToggle">
```

Reply:
You're right, but the other input fields and associated labels in `LayoutsSection` do not have it. Should I maybe remove it, or add and id for the other input fields in the section?

Reply:
Yeah add id for other if they are missing please.